aspnetcore/get-started.md

```diff
@@ -252,3 +252,11 @@ To learn more about ASP.NET Core, see the following:
 
 > [!div class="nextstepaction"]
 > <xref:index>
```

Comment:
```suggestion
> <xref:fundamentals/index>
```

Reply:
I thought we were also going to move the table of intro tutorials list here as well for folks that want to learn by doing?

Reply:
I don't recall it, but sure thing👍. I'll do that on the next commit.